bb/portmuxusr.c

```diff
@@ -206,7 +206,7 @@ int portmux_register(const char *app, const char *service, const char *instance)
 int portmux_deregister(const char *app, const char *service,
                        const char *instance)
 {
-    char name[NAMELEN];
+    char name[NAMELEN * 2];
```

Comment:
Why?

Reply:
Same as all the other instances of this fix: 
rc = snprintf(name, sizeof(name), "%s/%s/%s", app, service, instance);   
is definitely > 64 chars. Just making it easy and doing *2 rather than measuring the real app and service lengths.